motionplan/plan.go

```diff
@@ -307,3 +307,19 @@ func (e *ExecutionState) CurrentInputs() map[string][]referenceframe.Input {
 func (e *ExecutionState) CurrentPoses() map[string]*referenceframe.PoseInFrame {
 	return e.currentPose
 }
+
+// ErrorState returns the Pose between the planned Pose of the given component name its current Pose.
```

Comment:
This isn't correct.
`poses[e.index]` is where the base was expected to be when it started its arc, not where it is right now.

Reply:
Note that this will require knowledge of the framesystem

Reply:
yeah thats something I hadn't thought of yet.  Will have to rethink this.  Thanks for the callout